mmv1/third_party/terraform/tests/resource_sql_database_instance_test.go.erb

```diff
@@ -1274,6 +1274,42 @@ func TestAccSqlDatabaseInstance_SqlServerAuditConfig(t *testing.T) {
 	})
 }
 
+func TestAccSqlDatabaseInstance_Timezone(t *testing.T) {
```

Comment:
Since the timezone field is ForceNew, there's no need to have an update test for it. It just needs to be set by a test. Is there another SQL server test that you could add a hardcoded timezone field to? That would fulfill the requirement without adding a new test. Otherwise you could just eliminate the update portion of this test.

Reply:
I'll opt to remove the update portion of the test. If update fails, then it is working as intended.